dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/factory/impl/BitstreamResourcePolicyAddPatchOperation.java

```diff
@@ -74,17 +76,19 @@ void add(Context context, Request currentRequest, InProgressSubmission source, S
                         newAccessConditions.add(evaluateSingleObject((LateObjectEvaluator) value));
                     }
 
-                    for (ResourcePolicyRest newAccessCondition : newAccessConditions) {
+                    for (UploadAccessConditionDTO newAccessCondition : newAccessConditions) {
                         String name = newAccessCondition.getName();
                         String description = newAccessCondition.getDescription();
 
                         //TODO manage error on select group and eperson
                         Group group = null;
                         if (newAccessCondition.getGroupUUID() != null) {
+                           // UUID uuidGroup = UUID.fromString(newAccessCondition.getGroup().getUuid());
                             group = groupService.find(context, newAccessCondition.getGroupUUID());
                         }
                         EPerson eperson = null;
                         if (newAccessCondition.getEpersonUUID() != null) {
+                           // UUID uuidEPerson = UUID.fromString(newAccessCondition.getEperson().getUuid());
```

Comment:
There's two commented out lines here that don't seem to be necessary.  Here and 5 lines above this line.